client/app.js

```diff
@@ -0,0 +1,32 @@
+const form = document.querySelector('#new-blog-form');
+
+form.addEventListener('submit', submitPost);
+
+// create
+function submitPost(e){
+    e.preventDefault();
+
+    const blogData = {
+        title: e.target.title.value,
+        author: e.target.author.value,
+        content: e.target.content.value
+        
+    };
+
+    const options = { 
+        method: 'POST',
+        body: JSON.stringify(blogData),
+        headers: { "Content-Type": "application/json" }
+    };
+
+    fetch('http://localhost:3000/blog', options)
+        .then(r => r.json())
+        .then(showBlog)
+        .then(() => e.target.reset())
+        .catch(console.warn)
+};
+
+// helpers
+function showBlog(blogData){
+    console.log(blogData)
+};
```

Comment:
POST method looks alright and I just checked and it works. I filled out the form and clicked 'Publish' and then checked your server and it was stored in your db. So your client and server are connected, you just have to find a way to display the newly written blog. Me and Kai managed to display it on the same page on our repo but struggled all day to redirect users to a new one.